exposed-tests/build.gradle.kts

```diff
@@ -6,6 +6,10 @@ plugins {
     kotlin("jvm") apply true
 }
 
+kotlin {
+    jvmToolchain(8)
+}
+
```

Comment:
This is the only module that fails to build for me because there's a function in `StatisticsFunctionTests.kt` that uses `BigDecimal.sqrt()`, which is only available as of JDK 9. I had used it to avoid duplicating code in `calculateVariance()`.

Reply:
We can bump it here, it's not published